build/lib/topostats/grains.py

```diff
@@ -0,0 +1,307 @@
+"""Find grains in an image."""
```

Comment:
Not sure how this file got added but it can safely be removed.

The `build/` directory is for making package releases and we don't need to include it in the repository (I'll add it into `.gitignore` when I remove the `.png`).

Reply:
Ah I see whoops. I'm not sure how that happened.